pkg/apis/contrail/v1alpha1/keystone_types.go

```diff
@@ -25,9 +25,10 @@ type KeystoneConfiguration struct {
 // KeystoneStatus defines the observed state of Keystone
 // +k8s:openapi-gen=true
 type KeystoneStatus struct {
-	Active bool     `json:"active,omitempty"`
-	Port   int      `json:"port,omitempty"`
-	IPs    []string `json:"ips,omitempty"`
+	Active    bool     `json:"active,omitempty"`
+	Port      int      `json:"port,omitempty"`
+	IPs       []string `json:"ips,omitempty"`
+	ClusterIP string   `json:"clusterIP,omitempty"`
```

Comment:
Maybe let's name it `IPaddress` or just `IP`. ClusterIP name suggest that it is "ClusterIP" however it could be something different like LB IP or even something external

Reply:
Let's consider unifying IP address and a Port to one field `Endpoint`

Reply:
NA